packages/opentelemetry-resources/test/detectors/browser/EnvDetector.test.ts

```diff
@@ -14,76 +14,13 @@
  * limitations under the License.
  */
 
-import { RAW_ENVIRONMENT } from '@opentelemetry/core';
-import * as assert from 'assert';
 import { envDetector } from '../../../src';
 import { describeBrowser } from '../../util';
-import {
-  assertEmptyResource,
-  assertWebEngineResource,
-} from '../../util/resource-assertions';
+import { assertEmptyResource } from '../../util/resource-assertions';
 
 describeBrowser('envDetector() on web browser', () => {
-  describe('with valid env', () => {
-    before(() => {
-      (
-        globalThis as typeof globalThis & RAW_ENVIRONMENT
-      ).OTEL_RESOURCE_ATTRIBUTES =
-        'webengine.name="chromium",webengine.version="99",webengine.description="Chromium",custom.key="custom%20value"';
-    });
-
-    after(() => {
-      delete (globalThis as typeof globalThis & RAW_ENVIRONMENT)
-        .OTEL_RESOURCE_ATTRIBUTES;
-    });
-
-    it('should return resource information from environment variable', async () => {
-      const resource = envDetector.detect();
-      assert.ok(resource.attributes);
-      assertWebEngineResource(resource, {
-        name: 'chromium',
-        version: '99',
-        description: 'Chromium',
-      });
-      assert.strictEqual(resource.attributes['custom.key'], 'custom value');
-    });
-  });
-
-  describe('with invalid env', () => {
```

Comment:
with the new method is no longer possible to have invalid values?
or doesn't matter because it returns the empty and you're already testing that below?